DUI3-DX/Converters/Autocad/Speckle.Converters.AutocadShared/ToHost/Raw/AutocadPolycurveToHostPolylineRawConverter.cs

```diff
@@ -0,0 +1,53 @@
+using System.Collections.Generic;
+using Speckle.Converters.Autocad.Extensions;
+using Speckle.Converters.Common;
+using Speckle.Converters.Common.Objects;
+using Speckle.Core.Kits;
+
+namespace Speckle.Converters.Autocad2023.ToHost.Raw;
+
+public class AutocadPolycurveToHostPolylineRawConverter : IRawConversion<SOG.Autocad.AutocadPolycurve, ADB.Polyline>
+{
+  private readonly IRawConversion<SOG.Vector, AG.Vector3d> _vectorConverter;
+  private readonly IConversionContextStack<Document, ADB.UnitsValue> _contextStack;
+
+  public AutocadPolycurveToHostPolylineRawConverter(
+    IRawConversion<SOG.Vector, AG.Vector3d> vectorConverter,
+    IConversionContextStack<Document, ADB.UnitsValue> contextStack
+  )
+  {
+    _vectorConverter = vectorConverter;
+    _contextStack = contextStack;
+  }
+
+  public ADB.Polyline RawConvert(SOG.Autocad.AutocadPolycurve target)
+  {
+    if (target.normal is SOG.Vector normal && target.elevation is double elevation)
```

Comment:
I would throw first with single if block instead if-else. Not a big deal but it is nicer to do them similar as we throw first in conversions